lib/allthenews_app.dart

```diff
@@ -1,13 +1,18 @@
 import 'package:allthenews/ui/common/theme/theme.dart';
 import 'package:allthenews/ui/pages/news_list/news_list_page.dart';
+import 'package:allthenews/ui/pages/news_list/secondary_news/secondary_news_list_page.dart';
 import 'package:flutter/material.dart';
 
 class AllTheNewsApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      home: NewsListPage(),
       theme: newsTheme,
+      initialRoute: NewsListPage.id,
+      routes: {
+        NewsListPage.id: (context) => NewsListPage(),
+        SecondaryNewsListPage.id: (context) => SecondaryNewsListPage(),
+      },
```

Comment:
patrzyłem jak to jest w innych projektach to w jednym używają Fluro, a w drugim po prostu navigatora bez routów, nie wiem czy takie przetrzymywanie ID jako consty w każdym widoku jest jakimś best practices, ale wygląda okej :)

Reply:
generalnie przeglądałem różne projekty flutterowe i żadne z rozwiązań jakoś mi wybitnie nie przypadło do gustu szczerze mówiąc (zewnętrznych libek nie sprawdzałem), tu jedyną zaletą jest jakaś tam kontrola większa niż raw string :P